utils/local-devnet/service_node_network.py

```diff
@@ -116,14 +117,14 @@ def __init__(self, datadir, *, oxen_bin_dir, anvil_path, eth_sn_contracts_dir, s
                 break
 
         # Deploy smart contracts from eth-sn-contracts (if specified)
-        if eth_sn_contracts_dir is not None:
-            eth_sn_contracts_makefile_path = eth_sn_contracts_dir / 'Makefile'
-            if os.path.exists(eth_sn_contracts_makefile_path):
-                subprocess.run(['make', 'deploy-local-devnet'],
-                               cwd=eth_sn_contracts_dir,
-                               check=True)
-            else:
-                raise RuntimeError('eth-sn-contracts expected file to exist \'{}\' but does not. Exiting'.format(anvil_path))
+        if eth_sn_contracts_dir is None:
+            raise RuntimeError('eth-sn-contracts directory required')
```

Comment:
Looks sound, this one can be made a mandatory argument through argparse instead of a runtime exception